packages/block-editor/src/components/link-control/search-create-button.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { Button, Icon } from '@wordpress/components';
+
+export const LinkControlSearchCreate = ( {
+	searchTerm = '',
+	onClick,
+	itemProps,
+	isSelected,
+} ) => {
+	return (
+		<Button
+			{ ...itemProps }
+			className={ classnames(
+				'block-editor-link-control__search-create block-editor-link-control__search-item',
```

Comment:
I can see the `block-editor-link-control` class name prefix used a lot. Do you think we could move it to a constant?

E.g. in `constants.js`:

```javascript
export const blockName = 'block-editor-link-control';
```

So then we could

```javascript
import { blockName } from './constants';

<div className={ `${blockName}__search-item ` }>...
```

Reply:
Personally I feel ok about it being explicit without the additional abstraction.

Reply:
> I can see the `block-editor-link-control` class name prefix used a lot. Do you think we could move it to a constant?

I agree with @getdave . The repetition is not too concerning to me, at least not in contrast with the alternative. One other practical implication I have seen of this alternative is that it makes it unnecessarily difficult to search code when the search term would be expected to be the full concatenated string.